README.md

```diff
@@ -1,16 +1,20 @@
 # `nixfmt`
 
-`nixfmt` is a formatter for Nix code, intended to easily apply a uniform style.
-
-You are encouraged to test this out on your code and submit any undesirable formatting you find as an issue
+`nixfmt` is the official formatter for Nix language code, intended to easily apply a uniform style.
 
 ![Build Status](https://github.com/NixOS/nixfmt/actions/workflows/main.yml/badge.svg?branch=master)
 
 ## State
 
-`nixfmt` will form the basis for the initial official standard Nix formatter, as established by [RFC 166](https://github.com/NixOS/rfcs/pull/166).
+`nixfmt` was originally developed by [Serokell](https://github.com/serokell).
+It was used as the basis for the official standardised Nix formatter, as established by [RFC 166](https://github.com/NixOS/rfcs/pull/166).
```

Comment:
This feels slightly off. It reads as if the current project _was_ used as the basis of some other project that came out of rfc166.

I believe this could be clarified:
```suggestion
The original `nixfmt` formed the basis of [RFC 166](https://github.com/NixOS/rfcs/pull/166), which this project now implements.
```

or maybe:

```suggestion
The original `nixfmt` formed the basis of [RFC 166](https://github.com/NixOS/rfcs/pull/166), which this project will implement in the [upcoming release](https://github.com/NixOS/nixfmt/issues/272).
```